app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,74 @@
+class TweetsController < ApplicationController
+
+use Rack::Flash
+
+get "/tweets" do 
+  if logged_in?
+    @user = current_user
+    erb :'/tweets/tweets'
+  else
+    redirect "/login"
+  end
+end
+
+
+get '/tweets/new' do
+  if logged_in?
+    @user = current_user
+    erb :'/tweets/create_tweet'
+  else
+    redirect "/login"
+  end
+end
+
+post '/tweets' do
+  if !params[:content].empty?
+  current_user.tweets.create(content: params[:content], user_id: session[:user_id])
```

Comment:
great use of the associative create method command, but with this method you don't need to set user_id. This will automatically be done for you.